Source/ReactiveCore/MoyaError.swift

```diff
@@ -9,9 +9,10 @@ public enum MoyaError: ErrorType {
     case Underlying(ErrorType)
 }
 
-// If RACSignal support gets removed we can also remove everything under here
+@available(*, deprecated, message="This will be removed when ReactiveCocoa 4 becomes final. Please visit https://github.com/Moya/Moya/issues/298 for more information.")
 public let MoyaErrorDomain = "Moya"
 
+@available(*, deprecated, message="This will be removed when ReactiveCocoa 4 becomes final. Please visit https://github.com/Moya/Moya/issues/298 for more information.")
 public enum MoyaErrorCode: Int {
```

Comment:
If the Int is taken away then it is just what MoyaError is. I don't really see a use for both MoyaErrorCode and MoyaError when we remove RAC 2 support. The only use of MoyaErrorCode is to generate an NSError for RAC 2.